apps/docs/src/@primer/gatsby-theme-doctocat/components/wrap-page-element.js

```diff
@@ -1,15 +1,41 @@
 import React from 'react'
-import {BaseStyles} from '@primer/react'
+import {BaseStyles, themeGet} from '@primer/react'
 import {ColorThemeProvider} from '../../../components/content/ColorThemeContext'
+import {createGlobalStyle} from 'styled-components'
 import '../../../../../../packages/react/lib/css/main.css'
 import '../../../../../../packages/fonts/fonts.css'
 
-// Shadowing this Doctocat file
+// Doctocat global styles https://github.com/primer/doctocat/blob/main/theme/src/components/wrap-page-element.js
+const GlobalStyles = createGlobalStyle`
+  body {
+    color: ${themeGet('colors.fg.default')};
+    background-color: ${themeGet('colors.canvas.default')};
+  }
+
+  .footnotes {
+    font-size: ${themeGet('fontSizes.1')};
+    color: ${themeGet('colors.fg.subtle')};
+
+    ol {
+      padding-left: ${themeGet('space.3')};
+    }
 
+  .footnote-backref {
+      font-family: ${themeGet('fonts.mono')};
```

Comment:
We have our own Primer Brand tokens for this. Can we use those values instead and rely less on PRC for these styles? Particularly as they will be deprecating themeGet soon.

Reply:
Removed all `themeGet` in pro of tokens